javaslang/src/test/java/javaslang/control/TryTest.java

```diff
@@ -912,9 +911,9 @@ private RuntimeException error() {
 
     private static <T, X extends Throwable> Try<T> failure(Class<X> exceptionType) {
         try {
-            final X exception = exceptionType.newInstance();
+            final X exception = exceptionType.getConstructor().newInstance();
             return Try.failure(exception);
-        } catch (InstantiationException | IllegalAccessException x) {
+        } catch (InstantiationException | IllegalAccessException | NoSuchMethodException | InvocationTargetException ignored) {
```

Comment:
Thank you, I didn't know about that :).
Since this is part of the tests (i.e. it's not part of the public contract), we could simply catch a `Throwable`, since that's what we're testing.